src/ggrc/models/request.py

```diff
@@ -65,8 +65,8 @@ class Request(AutoStatusChangable, Assignable, Documentable, Personable,
   # TODO Make status via Stateful Mixin
   status = deferred(db.Column(db.Enum(*VALID_STATES), nullable=False,
                               default=tuple(START_STATE)[0]), 'Request')
-  requested_on = deferred(db.Column(db.Date, nullable=False), 'Request')
-  due_on = deferred(db.Column(db.Date, nullable=False), 'Request')
+  start_date = deferred(db.Column(db.Date, nullable=False), 'Request')
```

Comment:
@samotarnik didn't you already perform this change on the `release/candidate` branch? How did it turn back? Automerge maybe?